templates/parachain/pallets/template/src/lib.rs

```diff
@@ -66,17 +66,16 @@ mod benchmarking;
 // To see a full list of `pallet` macros and their use cases, see:
 // <https://paritytech.github.io/polkadot-sdk/master/pallet_example_kitchensink/index.html>
 // <https://paritytech.github.io/polkadot-sdk/master/frame_support/pallet_macros/index.html>
-#[frame_support::pallet]
+#[frame::pallet]
 pub mod pallet {
-	use frame_support::{dispatch::DispatchResultWithPostInfo, pallet_prelude::*, DefaultNoBound};
-	use frame_system::pallet_prelude::*;
-	use sp_runtime::traits::{CheckedAdd, One};
+	use frame::{
+		arithmetic::{CheckedAdd, One},
```

Comment:
We could also add these to the prelude, in effect by adding them to the `frame_support::pallet_prelude`. Both seem quite reasonable to be there.

Reply:
Done